File: Uncategorized/7. Reverse Integer/7. Reverse Integer.js
var reverse = function(x) {
    let number = x.toString();
    let isNegative;
    if (number[0]=='-'){
        isNegative=true;
        number.slice(1);
    }
    else {
        isNegative=false;
    }
    number=number.split("").reverse().join("");
    x=parseInt(number);
    if(x>=Math.pow(2,31)){
        return 0;
    }
    return isNegative?x*(-1):x;
};

File: Uncategorized/7. Reverse Integer/7. Reverse Integer.py
class Solution:
    def reverse(self, x: int) -> int:
        number = str(x)
        if number[0]=='-':
            isNegative=True
            number=number[1:]
        else:
            isNegative=False
        number=number[::-1]
        x=int(number)
        if x >= 2**31:
            return 0
        return x*(-1) if isNegative else x 